content/en/infrastructure/livecontainers.md

```diff
@@ -13,73 +13,100 @@ further_reading:
-   ...
-{{< /code-block >}}
-
-**Note**: The cluster name must be 80-characters or less.
-
-On Google's GKE, AWS EKS, and Azure AKS, this is unnecessary, unless the Agent and the cluster Agent don't have access to the cloud metadata APIs, or the cluster name is longer than 80 characters.
+The Agent and [Cluster Agent][5] can be configured to retrieve Kubernetes resources for [Live Containers][6]. This feature allows you to monitor the state of pods, deployments and other Kubernetes concepts in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+
+Kubernetes resources for Live Containers requires [Agent version >= 7.21.1][7] prior to the configurations below.
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+If you are using the official [Datadog Helm Chart][1]:
+
+- Use chart version 2.4.5 or above
+- Set `datadog.orchestratorExplorer.enabled` to true in [values.yaml][2]
+- Deploy a new release
+
+
+[1]: https://github.com/DataDog/helm-charts
+[2]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
+{{% /tab %}}
+{{% tab "DaemonSet" %}}
+
+To configure a DaemonSet, [Cluster Agent][1] version >= 1.9.0 is required. The Cluster Agent must be running, and the Agent must be able to communicate with it. See the [Cluster Agent Setup documentation][2] for configuration.
+
+1. Set the Cluster Agent container with the following environment variable:
+
+      ```
+      - name: DD_ORCHESTRATOR_EXPLORER_ENABLED
+        value: "true"
+      ```
+
+2. Set the Cluster Agent ClusterRole with the following RBAC permissions:
+
+    ```
+      ClusterRole:
+      - apiGroups:  # To create the datadog-cluster-id CM
+        - ""
+        resources:
+        - configmaps
+        verbs:
+        - create
+        - get
+        - update
+      ...
+      - apiGroups:  # Required to get the kube-system namespace UID and generate a cluster ID
+        - ""
+        resources:
+        - namespaces
+        verbs:
+        - get
+      ...
+      - apiGroups:  # to collect new resource types
+        - "apps"
+        resources:
+        - deployments
+        - replicasets
+        verbs:
+        - list
+        - get
+        - watch
+      ```
+
+    These permissions are needed in order to create a `datadog-cluster-id` ConfigMap in the same Namespace as the Agent DaemonSet and the Cluster Agent Deployment, as well as to collect Deployments and ReplicaSets.
+
+    If the `cluster-id` ConfigMap doesn't get created by the Cluster Agent, the Agent pod will not start, and fall in `CreateContainerConfigError` status. If the Agent pod is stuck because this ConfigMap doesn't exist, update the Cluster Agent permissions and restart its pods to let it create the ConfigMap and the Agent pod will recover automatically.
+
+3. The Process-agent must be running (it doesn't need to run the process collection), and configured with the following options:
```

Comment:
This step, unlike the previous two, is to be applied to the DaemonSet. That's because the process-agent runs in the agent DaemonSet, not in the Cluster Agent Deployment. We might also want to point this out.